tibanna/cw_utils.py

```diff
@@ -324,19 +324,117 @@ def max_ebs_read_used_all_pts(self):
         pts = [(r['Average'], r['Timestamp']) for r in res['Datapoints']]
         return[p[0] for p in sorted(pts, key=lambda x: x[1])]
 
+    
+    @staticmethod
+    def extract_metrics_data(file_contents):
```

Comment:
I would strongly recommend unit testing this function especially. You can probably just pull down a sample metrics file to process.